devices/sinope.js

```diff
@@ -987,7 +1085,7 @@ module.exports = [
         },
     },
     {
-        zigbeeModel: ['SW2500ZB'],
+        zigbeeModel: ['SW2500ZA'],
```

Comment:
I don't understand this change, this will make the `SW2500ZB` unsupported.

Reply:
Koen, sorry and thanks for catching that.  I had no intention to rename the zigbeeModel.  I use those locally as this is the only way I found to test my modifications.  I have to use a different name in the external converter (and in database.db) than the one in the official load. Thanks!